WORKSPACE

```diff
@@ -0,0 +1,50 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+android_sdk_repository(
+    name = "androidsdk",
+    api_level = 28,
```

Comment:
How do we coordinate this version with the one defined in the manifest?

Reply:
Since the manifest doesn't have an explicit `<uses-sdk>` tag - how do I know what version it is using? Maybe we can add the tag, define a constant for the version number, and use it in both the WORKSPACE and manifest? I haven't seen anything in Bazel documentation talking about this issue but I will see if there is a simpler way to do this.

Reply:
I am adding this point to our 1:1 doc to discuss further!

Reply:
Can you file an issue & add a TODO to track this so that the conversation can be resolved? I don't think we need to block the PR on this point, but we should account for it.

Reply:
Sure!

Reply:
Thanks!